libraries/chain/son_wallet_evaluator.cpp

```diff
@@ -0,0 +1,69 @@
+#include <graphene/chain/son_wallet_evaluator.hpp>
+
+#include <graphene/chain/database.hpp>
+#include <graphene/chain/son_wallet_object.hpp>
+
+namespace graphene { namespace chain {
+
+void_result create_son_wallet_evaluator::do_evaluate(const son_wallet_create_operation& op)
+{ try{
```

Comment:
payer should be the SON consensus account get_son_btc_account_id() ?

Reply:
Resolved, line 15.